test/image.bats

```diff
@@ -457,7 +457,7 @@ EOF
 		| .image.image = $image' \
 		"$TESTDATA"/container_config.json > "$TESTDIR"/memory.json
 
-	run crictl run "$TESTDIR"/memory.json "$TESTDATA"/sandbox_config.json
+	run ! crictl run "$TESTDIR"/memory.json "$TESTDATA"/sandbox_config.json
```

Comment:
I think there is a typo or something; this test is supposed to run [a container and doesn't fail.](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/cri-o_cri-o/8057/pull-ci-cri-o-cri-o-main-ci-fedora-kata/1782433161666367488/artifacts/fedora-kata/cri-o-gather/artifacts/testout.txt#:~:text=alpine%40sha256%3A65b3a80ebe7471beecbc090c5b2cdd0aafeaefa0715f8f12e40dc918a3a70e32-,%23%20f0b894c92ead47bdeffc3e64066d8631b22bfa7edc679156b8843283acf82d6f,-%23%20%23%20%2D%2D%2D%20crio.log%20%3A%3A%20%2D%2D%2D%0A%23%20time) 

`container_min_memory` is `"17.5MiB"` and it is enough to launch a container.

Reply:
container_min_memory is a minimum memory so the container memory should be above the minimum, right?
https://github.com/cri-o/cri-o/blob/9bac613f5fd24c3d690dfb2233f600d157e94baa/internal/config/cgmgr/cgmgr_linux.go#L149-L155

It sets 12MiB as a container memory so it should be failed, shouldn't it?

https://github.com/cri-o/cri-o/blob/c65ecf08ccd32b90e4fcf97b8f5c46418904c506/test/image.bats#L456

Reply:
This function does the check here:
https://github.com/cri-o/cri-o/blob/d90763d245d0756977cf4db0f9b8c0bbf2796f8f/server/container_create_linux.go#L434-L436 

- `memoryLimit` - `resources.memory_limit_in_bytes` memory requested for container
- `minMemory`   - `container_min_memory` minimum memory needs to run a container

```go
func VerifyMemoryIsEnough(memoryLimit /* 12MiB */, minMemory /* 17MiB */) error { 
  if 12MiB < 17MiB {
    return error
  }
  // ...
}
```

Hence, if `memoryLimit` is above `minMemory`, the container can't start.

Reply:
@cri-o/cri-o-maintainers ^^^ WDYT?

Reply:
sorry I can't still get it.
12 MiB < 17 MiB is true, so `VerifyMemoryIsEnough` should return an error and the container can't start...?

the same as https://github.com/cri-o/cri-o/pull/7832/files#diff-6391e62911e38278fada73fcfa52f4480e77cb2aea66483934da7e66871dddb5

Reply:
Oh, I see, you're right `run ! ...` is the right assertion here.